### ISSUE TITLE
Any.Compute<unknown> resolves to {}

### ISSUE THREAD

**Issue by kulla** [millsp/ts-toolbelt#168]
#### Describe the bug
`Any.Compute<unknown>` resolves to `{}` instead of `unknown`

#### Reproduce the bug
File `test.ts`:

```ts
import { Any } from 'ts-toolbelt'

// Throws error: Type 'null' is not assignable to type '{}'
const a: Any.Compute<unknown> = null

// This in contrast works since null is applicable to unknown
const b: unknown = null
```

Error when running `tsc`:

```
test.ts:3:7 - error TS2322: Type 'null' is not assignable to type '{}'.

3 const a: Any.Compute<unknown> = null
        ~
```

File `tsconfig.json`:

```json
{
  "compilerOptions": {
    "strictNullChecks": true,
    "strict": true,
    "lib": ["es6", "dom"]
  }
}
```

File `package-lock.json`:

```json
{
  "name": "test",
  "version": "1.0.0",
  "lockfileVersion": 1,
  "requires": true,
  "dependencies": {
    "ts-toolbelt": {
      "version": "8.4.0",
      "resolved": "https://registry.npmjs.org/ts-toolbelt/-/ts-toolbelt-8.4.0.tgz",
      "integrity": "sha512-hnGJXIgC49ZuF5g5oDthoge8t4cvT0dYg2pYO5C6yV/HmUUy1koooU2U/5K2N+Uw++hcXQpJAToLRa+GRp28Sg=="
    },
    "typescript": {
      "version": "4.2.0-dev.20201229",
      "resolved": "https://registry.npmjs.org/typescript/-/typescript-4.2.0-dev.20201229.tgz",
      "integrity": "sha512-LbUYLWEV1RrduLTYuUaduTT0HXqdJ4jbs2rdUzwFFIEPdRsnRbWCCdSV0OgA7b9O6n3dwMP7VLqoBnqUd/RByw=="
    }
  }
}
```

#### Expected behavior
`tsc` should run without an error.

#### Additional context
I ran into this issue while making a PR for https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/ramda There I noticed that `Any.Compute<{ foo: unknown }>` became `{ foo: {} }` instead of `{ foo: unknown }`.

**Comment by millsp** [comment]
Hey @kulla, thanks for reporting this, it will be scheduled for the next release

**Comment by kulla** [comment]
> Hey @kulla, thanks for reporting this, it will be scheduled for the next release

Thank's. Just a comment: Looking at https://github.com/millsp/ts-toolbelt/blob/master/tst/Any.ts#L31-L34 I see that there are no tests currently for `Compute<...>`. However this is possible with https://github.com/microsoft/dtslint since they have the assertation `$ExpectType`. This is how I found this bug, since I used `Compute` in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50331 (I replaced it with `Any.Clean<..>` which does not have this issue).

However, thanks for this library! :heart: Before my PR in DefinitelyTyped i didn't know it and it seems to be really useful. In case I will have free time I would like to contribute.
